Method,Name,Error,Occurrences
GET,/Categorias,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",5
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 7'),447
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 6'),586
GET,/Productos,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",9
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 3'),399
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 8'),624
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 2'),431
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 3'),414
GET,/Categorias,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",40
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 4'),465
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 7'),384
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 1'),431
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 4'),450
GET,/Pedidos/5925,HTTPError('500 Server Error: Internal Server Error for url: /Pedidos/5925'),3
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 12'),475
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 11'),406
GET,/Mesas,RemoteDisconnected('Remote end closed connection without response'),649
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 9'),443
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 3'),434
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 7'),388
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 9'),585
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 9'),440
GET,/Pedidos/5925,RemoteDisconnected('Remote end closed connection without response'),1911
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 12'),491
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 2'),591
GET,/Mesas,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",10
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 11'),418
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 3'),436
GET,/Pedidos/5925,"ConnectionResetError(10054, 'Se ha forzado la interrupción de una conexión existente por el host remoto', None, 10054, None)",3
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 9'),435
GET,/Categorias,HTTPError('500 Server Error: Internal Server Error for url: /Categorias'),5
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 7'),626
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 1'),436
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 10'),657
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 13'),409
GET,/Mesas,HTTPError('500 Server Error: Internal Server Error for url: /Mesas'),2
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 8'),420
GET,/Pedidos/5925,"ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x....>, 'Connection to 192.168.100.15 timed out. (connect timeout=None)')",39
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 1'),399
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 12'),449
GET,/Categorias,RemoteDisconnected('Remote end closed connection without response'),2569
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 2'),460
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 13'),391
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 5'),385
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 8'),413
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 5'),388
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 5'),352
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 1'),415
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 12'),443
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 9'),440
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 10'),399
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 1'),620
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 2'),388
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 8'),446
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 4'),575
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 11'),413
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 2'),469
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 7'),411
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 3'),593
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 5'),617
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 13'),450
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 6'),391
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 9'),460
PUT,Abrir Mesa,CatchResponseError('Fallo al abrir mesa 4'),391
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 11'),412
GET,/Productos,RemoteDisconnected('Remote end closed connection without response'),5150
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 11'),379
POST,Agregar Item,CatchResponseError('Fallo al agregar item 1 en mesa 13'),396
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 4'),382
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 2'),420
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 5'),398
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 8'),439
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 1'),408
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 5'),440
POST,Agregar Item,CatchResponseError('Fallo al agregar item 3 en mesa 8'),413
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 4'),447
PUT,Cerrar Mesa,CatchResponseError('Fallo al cerrar mesa 11'),583
POST,Agregar Item,CatchResponseError('Fallo al agregar item 4 en mesa 3'),456
POST,Agregar Item,CatchResponseError('Fallo al agregar item 2 en mesa 7'),371
